src/main/java/com/coveo/spillway/limit/LimitBuilder.java

```diff
@@ -92,30 +99,42 @@ private LimitBuilder() {}
    * Adds a call back that will be called when the specified limit (using {@link #to(int)}) is reached.
    *
    * @param limitTriggerCallback The callback {@link LimitTriggerCallback}
-   * @return The current LimitBuilder
+   * @return The current {@link LimitBuilder}
    */
   public LimitBuilder<T> withExceededCallback(LimitTriggerCallback limitTriggerCallback) {
     triggers.add(new ValueThresholdTrigger(limitCapacity, limitTriggerCallback));
     return this;
   }
 
   /**
+   * Adds a {@link LimitOverride} to the current {@link Limit}.
+   *
+   * @param limitOverride The {@link LimitOverride}
+   * @return The current {@link LimitBuilder}
+   */
+  public LimitBuilder<T> withLimitOverride(LimitOverride limitOverride) {
```

Comment:
Yes exactly, as you can see I redefined the equals to match only the property value.
I thinks its reasonable to assume that we should only have one override for each property.